scripts/pi-hole/php/header.php

```diff
@@ -333,12 +334,7 @@ function pidofFTL()
                 <div class="pull-left info">
                     <p>Status</p>
                     <?php
-                    $pistatus = pihole_execute('status web');
-                    if (isset($pistatus[0])) {
-                        $pistatus = intval($pistatus[0]);
-                    } else {
-                        $pistatus = null;
-                    }
+                    $pistatus = piholeStatus();
```

Comment:
I'm fine with your code.

I think you can remove the test for `null` on line 344

```php
} elseif ($pistatus == -2 || is_null($pistatus)) {
```

```php
} elseif ($pistatus == -2) {
```